lib/tasks/local_index.rake

```diff
@@ -9,7 +9,7 @@ task :local_index => :environment do
 
   Dir.chdir(dir) do
     # find all tags
-    tags = `git tag | egrep 'v1|v2'`.strip.split("\n")
+    tags = `git tag | egrep 'v2\.8'`.strip.split("\n")
```

Comment:
What's this hunk doing in here?

Is it just a leftover from debugging, convincing the rakefile to only process recent versions, since doing them all takes so long (I ask because I wrote the _same_ hunk while looking at #749).